extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -0,0 +1,28 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-sidebar"></a> `amp-sidebar`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A sidebar provides a way to display meta content intended for temporary access (navigation links, buttons, menus, etc.).The sidebar can be revealed by a button tap while the main content remains visually underneath.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Work in progress.</td>
```

Comment:
Let's keep "work in progress". But since experiment is already defined, let's also add the experimental part here as well. You can split them all with `div`s - that'd be just fine.